doc/source/developer_guide/dg_tools.rst

```diff
@@ -0,0 +1,151 @@
+
+There is a Fortran program in **configuration/tools/cice4_restart_conversion**
+that will help convert a CICE4 restart file into a CICE5 restart file.
+There is a bit of documentation contained in that source code about how
+to build, use, and run the tool.  A few prognostic variables were changed
+from CICE4 to CICE5 which fundamentally altered the fields saved to
+the restart file.  See 
+**configuration/tools/cice4_restart_conversion/convert_restarts.f90** 
+for additional information.
+
+
+.. _jra55datasettool:
+
+JRA55 forcing datasets
+------------------------
+
+This section describes how to generate JRA55 forcing data for the CICE model.
+Raw JRA55 files have to be interpolated and processed into input files specifically
+for the CICE model.  A tool exists in **configuration/tools/jra55_datasets**
+to support that process.
+The raw JRA55 data is obtained from the NCAR/UCAR Research Data Archive and
+the conversion tools are written in python.
+
+Requirements
+*********************
+
+Python3 is required, and the following
+python packages are required with the tested version number in parenthesis.  These
+versions are not necessarily the only versions that work, they just indicate what 
+versions were used when the script was recently run.
+
+- python3 (python3.7.9)
+- numpy (1.18.5)
+- netCDF4 (1.5.5)
+- ESMPy (8.0.0)
+- xesmf (0.3.0)
+- argparse
+- datetime
+
+NCO is required for aggregating the output files into yearly files.
+
+- netcdf (4.7.4)
+- nco (4.9.5)
+
+Raw JRA55 forcing data
+*************************
+
+The raw JRA55 forcing data is obtained from the UCAR/NCAR Research Data Archive,
+https://rda.ucar.edu/.  You must first register (free) and then signin.  The
+"JRA-55 Reanalysis Daily 3-Hourly and 6-Hourly Data" is ds628.0 and can be found here,
+https://rda.ucar.edu/datasets/ds628.0.  
+
+The "Data access" tabs will provide a list of product categories.
+The JRA55 data of interest are located in 2 separate products. Winds, air 
+temperature, and specific humidity on the model grid are included in "JRA-55 
+3-Hourly Model Resolution 2-Dimensional Instantaneous Diagnostic Fields". 
+Precipitation and downward radiation fluxes are found in "JRA-55 3-Hourly 
+Model Resolution 2-Dimensional Average Diagnostic Fields".  (Note the 
+difference between instantaneous and averaged data products. There are several 
+JRA55 datasets available, you will likely have to scroll down the page to find 
+these datasets.) Data are also available on a coarser 1.25° grid, but the tools
+are best used with the native TL319 JRA55 grid.
+
+The fields needed for CICE are
+
+- specific humidity (3-hourly instantaneous), Qa
+- temperature (3-hourly instantaneous), Tair
+- u-component of wind (3-hourly instantaneous), uatm
+- v-component of wind(3-hourly instantaneous), vatm
+- downward longwave radiation flux (3 hourly average), flw
+- downward solar radiation flux (3 hourly average), fsw
+- total precipitation (3 hourly average), fsnow
```

Comment:
Is JRA55 precipitation data liquid water equivalent?  Does it have to be converted to snow assuming some snow and liquid water densities, and if so, do we do that in our scripts?  It would be good to include the data units here.

Reply:
I had a look.  The total precip is treated as snow in the model regardless of air temperature or season.  There is no disaggregation into rain and snow based on temperature or anything else.  I added a note in the documentation about this.  I will not judge whether this is reasonable or not.